mysql-test/suite/galera/t/MDEV-33997.test

```diff
@@ -0,0 +1,48 @@
+--source include/galera_cluster.inc
+--source include/have_partition.inc
+
+#
+# MDEV-33997: Assertion `((WSREP_PROVIDER_EXISTS_ && this->variables.wsrep_on) && wsrep_emulate_bin_log) || mysql_bin_log.is_open()' failed in int THD::binlog_write_row(TABLE*, bool, const uchar*)
+#
+SET SESSION wsrep_osu_method=RSU;
+SET autocommit=0;
+
+CREATE TABLE t (c INT) PARTITION BY KEY(c) PARTITIONS 2;
+INSERT INTO t VALUES (1);
+INSERT INTO t SELECT 1 ;
+SELECT * FROM t;
+DROP TABLE t;
+
+#
+# MDEV-27296 : Assertion `((thd && (WSREP_PROVIDER_EXISTS_ && thd->variables.wsrep_on)) && wsrep_emulate_bin_log) || mysql_bin_log.is_open()' failed
+# Second test case
+#
+SET autocommit=1;
+SET SESSION wsrep_osu_method=RSU;
+CREATE TABLE t (c INT) PARTITION BY KEY(c) PARTITIONS 2;
+INSERT INTO t SELECT 1 ;
+SELECT * FROM t;
+DROP TABLE t;
+SET SESSION wsrep_osu_method=TOI;
+#
+# In case of log-bin INSERT will cause inconsistency on second node because table t
```

Comment:
If it is an intention to have this test in galera suite, the test should be fixed so that:
* The node_1 waits for node_2 to leave the cluster after inconsistency.
* Prefer `shutdown_mysqld.inc` over `kill_galera.inc` if possible.

Reply:
I moved this under wsrep suite.